From 7ccad88d873df45136dec4fe90e5475f42ce167c Mon Sep 17 00:00:00 2001
From: dmeier <dmeier@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Fri, 23 Sep 2016 08:02:00 +0000
Subject: [PATCH 228/462] MANTIS 42818 Devicehandler for TTS (text to speech)
 shall use MS Speech Platform - Version 11

Set new environment variables to Speech platform SDK

git-svn-id: svn://localhost/SelfServiceBuild/trunk@345 2eb42a87-3827-0410-b767-dc15055982e4
---
 SelfServiceBuild/Build/scripts/setenv.bat | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/SelfServiceBuild/Build/scripts/setenv.bat b/SelfServiceBuild/Build/scripts/setenv.bat
index ff9d4b9..ed5a91b 100644
--- a/SelfServiceBuild/Build/scripts/setenv.bat
+++ b/SelfServiceBuild/Build/scripts/setenv.bat
@@ -386,6 +386,10 @@ set DIR_MAGICK633=%DIR_GLOBALSLIB%\imagemagick\6.3.3-4-Q8\win32
 set INC_MAGICK633=-I%DIR_MAGICK633%\inc
 set LIB_MAGICK633=%DIR_MAGICK633%\lib\CORE_RL_magick_.lib %DIR_MAGICK633%\lib\CORE_RL_wand_.lib %DIR_MAGICK633%\lib\X11.lib
 
+set DIR_SPEECH11=%DIR_GLOBALSLIB%\Microsoft_SDKs\Speech\v11.0
+set INC_SPEECH11=-I%DIR_SPEECH11%\Include
+set LIB_SPEECH11=%DIR_SPEECH11%\Lib\sapi.lib
+
 set DIR_QTSDK=%DIR_GLOBALSLIB%\qtsdk\6.0\win32
 set INC_QTSDK=-I%DIR_QTSDK%\cincludes
 set LIB_QTSDK=%DIR_QTSDK%\libraries\qtmlclient.lib
-- 
2.41.0.windows.1

